### DISCUSSION TITLE
Migration Guide to migrate from Airtable to NocoDB

### DISCUSSION THREAD

**Discussion by nilsreichardt** [nocodb/nocodb#1765] [General]
It would be helpful to have some kind of migration guide that explains how to migrate from Airtable to NocoDB.

**Comment by o1lab** [comment]
@nilsreichardt : some of our users has successfully used sequin.io to migrate data from airtable to postgres. And then simply connect nocodb to that postgres. We have our own coming up in the pipeline as well.

**Reply by ewingrj** [reply]
ETA on yours? The challenge I'm having is linked records don't transfer over nicely when using sequin. All the airtable export tools export it as an array of record ids, but nocodb uses join tables

**Comment by o1lab** [comment]
Importing Airtable to NocoDB is available with latest version (0.91) - below is a simple demo

https://user-images.githubusercontent.com/5435402/169849198-f5840949-76c8-4434-9ef3-f9007a164660.mov

**Reply by nilsreichardt** [reply]
Great! ❤️
